### ISSUE TITLE
dev-7 regression, buffer uri isn't relative

### ISSUE THREAD

**Issue by fabrobinet** [KhronosGroup/glTF#286]
I saw this is generated (that shows up an absolute) when I expected just a relative uri
I got this:
`"uri": "/Users/fabricerobinet/Sites/viewer/glTF-webgl-viewer/model/duck/duck.bin"`
But expected result was simply (since both the bin and the glTF file are at the same level):
` "uri" : "duck.bin"

**Comment by javagl** [comment]
Was this supposed to be fixed, or closed for any other reason? 

I'm asking because I also observed this behavior: When the output path is *not* a single file name (i.e. when the output is supposed to be written to a different directory than where COLLADA2GLTF is called), then the URIs contain the whole output path. This bugged me a bit, and I recently saw in https://github.com/KhronosGroup/glTF/commit/72f0654acea78da20d511bc2c5beacd399152c4c that others seem to have this problem, too. 

I know, the handling of file paths and URIs and the platform dependencies here are a hassle in C++, but wonder whether this couldn't be fixed nevertheless...
